httpx/_cli/cli.py

```diff
@@ -0,0 +1,310 @@
+import json
+import sys
+import typing
+
+import click
+import rich.console
+import rich.progress
+import rich.syntax
+
+import httpx
+
+from .utils import (
+    format_request_headers,
+    format_response_headers,
+    get_download_filename,
+    get_lexer_for_response,
+)
+
+
+def print_request_headers(request: httpx.Request) -> None:
+    console = rich.console.Console()
+    http_text = format_request_headers(request)
+    syntax = rich.syntax.Syntax(http_text, "http", theme="ansi_dark", word_wrap=True)
```

Comment:
Any particular reason for going with `theme="ansi_dark"` instead of `background_color="default"`?

Reply:
Either the `ansi_light` and `ansi_dark` themes render okay on both light and dark console backgrounds...

<img width="568" alt="Screenshot 2020-09-22 at 09 19 56" src="https://user-images.githubusercontent.com/647359/93858907-cead5400-fcb4-11ea-927f-55a80a3d90b7.png">

<img width="466" alt="Screenshot 2020-09-22 at 09 20 02" src="https://user-images.githubusercontent.com/647359/93858915-d0771780-fcb4-11ea-8138-18df51606a19.png">

Using `background_color="default"` is really nice with `monokai` on dark consoles, but terrible on light backgrounds...

<img width="560" alt="Screenshot 2020-09-22 at 09 18 19" src="https://user-images.githubusercontent.com/647359/93858860-b63d3980-fcb4-11ea-98fc-90a8b6960fe9.png">

<img width="490" alt="Screenshot 2020-09-22 at 09 18 05" src="https://user-images.githubusercontent.com/647359/93858863-b9d0c080-fcb4-11ea-843d-4e89311cf25e.png">

Reply:
Gotcha. I have a dark terminal, so I preferred the default theme. Perhaps adding a `--theme={default,dark,light}` option could be an thing?